editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/FusePropertySection.java

```diff
@@ -674,6 +675,23 @@ public void modifyText(ModifyEvent e) {
 			txtField.setLayoutData(createPropertyFieldLayoutData());
 			c = txtField;
 
+			// grand Children inside a list of nodes
+		} else if(camelModelElement != null && camelModelElement.getParameter(p.getName()) instanceof List){
+			//TODO: use something better than a text for a list of elements as we have the obvious limitation of not supporting the ","
+			List<String> initialValue = (List<String>)camelModelElement.getParameter(p.getName());
+			String delimiter = ",";
+			String collect = initialValue.stream().collect(Collectors.joining(delimiter));
+			Text txtField = getWidgetFactory().createText(parent, collect, SWT.SINGLE | SWT.LEFT);
+			txtField.addModifyListener(new ModifyListener() {
+				@Override
+				public void modifyText(ModifyEvent e) {
+					Text txt = (Text) e.getSource();
+					camelModelElement.setParameter(p.getName(), Arrays.asList(txt.getText().split(delimiter)));
```

Comment:
if rare, it means that we introduce a loss of data to the user
if not existent, that's better to trim.
As we don't know i prefer to avoid the loss of data